repose-aggregator/components/filters/valkyrie-authorization-filter/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -63,13 +62,13 @@ class ValkyrieAuthorizationFilterTest extends FunSpec with BeforeAndAfterEach wi
   //todo: I suspect some of these tests are repetitive now, although they test it from different perspectives so
   // probably still worthwhile. I think some describe mocking behavior where it's not necessary as well. Short
   // timelines mean i can't dig into them right now.
-  val akkaServiceClient = mock[AkkaServiceClient]
-  val akkaServiceClientFactory = mock[AkkaServiceClientFactory]
-  val mockDatastoreService = mock[DatastoreService]
+  val akkaServiceClient: AkkaServiceClient = mock[AkkaServiceClient]
```

Comment:
They were flags by the static analysis tool. The rule is if it is a public value, then it requires a type; otherwise it shouldn't have one.